study.md

```diff
@@ -21,7 +21,7 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+To keep the code DRY.
 ```
```

Comment:
It can be, more specifically it's often used to avoid amguity by making sure we are referencing a specific object, and not a global object (e.g `window` in a web browser)